draft-poidt-ccamp-actn-poi-pluggable.md

```diff
@@ -147,42 +166,51 @@ informative:
    The full automation of the multilayer/multidomain network is a topic
    of high importance in the industry and the service providers
    community.  Typically, the layers composing such network are the IP/
-   MPLS, (with Segment Routing) and the Optical ones.  The requirements
+   MPLS packet domain (with Segment Routing) and the Optical domain 
+   providing DWDM transmission and photonic switching.  The requirements
    of high bandwidth availability and dynamic control of the networks
-   are of capital importance too.  The draft-ietf-teas-actn-poi-
-   applicability specifies very well how to control and manage
+   are of capital importance too.  
+   The {{!I-D.draft-ietf-teas-actn-poi-applicability}} 
+   specifies very well how to control and manage
    multilayer/multidomain networks using the Abstraction and Control of
    TE Networks (ACTN) architecture, see also {{my_figure-0}}.
 
-   New DWDM Coherent pluggable optics, such as ZR {{OIF-400ZR-01-0}} and
-   ZR+ {{Open_ZR-Plus_MSA}}, are enabling new multilayer network use cases
-   where the DWDM interface is located within the packet domain
-   equipment instead of being part of the Optical domain {{my_figure-1}}.
+   New pluggable 
+
+   Coherent pluggable digital coherent optics, such as ZR 
+   {{OIF-400ZR-01-0}} and ZR+ {{Open_ZR-Plus_MSA}}, are enabling new 
+   multilayer network use cases where the DWDM optical transmission
+   function is located within the packet domain equipment instead of 
```

Comment:
We need to be very careful with the term "domain". From my viewpoint, the "packet domain" is concerned with packet processing and not photonics and hence an equipment that has photonics is NOT in the packet domain, it straddles the two domains. A device straddling multiple domains is well recognized as there are many cases where a device spreads across multiple domains (admin, protection, technology etc.).

Reply:
v02

Reply:
Whilst I understand that there is significant work in adjustment to the notion of domains, I think the current rationale is flawed. I would minimally want to see some acknowledgement that the photonics is not really in the packet domain and that some refinement is required here.